Commands/Remove title number.plist

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>beforeRunningCommand</key>
+	<string>nop</string>
+	<key>command</key>
+	<string>#!/usr/bin/env ruby18 -wKU
+class Order
+
+  def initialize
+    @hh = [0] * 6  # html H1~H6
+  end
+
+  def maxl
+    @hh.size
+  end
+
+end
+
+
+data = $stdin.read
+order = Order.new
+
+data2 = data.gsub(/^(#+)([^#\n]+)/) do |x|
+  hx, title = $1, $2.strip
+  next x if hx.size &gt; order.maxl
+
+  title.sub! /^(\d+\.)+\s+/, ''
```

Comment:
I think this should be `/^[\d.]+\s+/ to support section numbers with and without trailing period (I personally do not add a trailing period when I enumerate sections)